src/DataCollector/DoctrineDataCollector.php

```diff
@@ -114,6 +119,10 @@ public function collect(Request $request, Response $response, Throwable|null $ex
                 }
             }
 
+            foreach ($em->getUnitOfWork()->getIdentityMap() as $className => $entityList) {
+                $entityCounts[$name][$className] = ($entityCounts[$name][$className] ?? 0) + count($entityList);
```

Comment:
```suggestion
                $entityCounts[$name][$className] = count($entityList);
```

We won't have the same key twice for each entity manager.